src/ducks/resource.js

```diff
@@ -33,75 +33,58 @@ const resourceReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchResource = (id, type) => {
+function fetchResource(id, type) {
   type = type || 'projects';
-  return (dispatch) => {
+  return (dispatch, getState) => {
+    const project = getState().project.data;
```

Comment:
FYI to make it clear: `connect` and `mapStateToProps` aren't vanilla Redux. They come from `react-redux`: https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options